app/utils/theme.js

```diff
@@ -27,3 +30,24 @@ export function setNavigatorStyles(navigator, theme) {
         screenBackgroundColor: theme.centerChannelBg,
     });
 }
+
+export function getAllowedThemes(state) {
+    const config = getConfig(state);
+    const activeTheme = getTheme(state);
+    const allThemes = Object.keys(Preferences.THEMES).map((key) => ({
+        ...Preferences.THEMES[key],
+        key,
+    }));
+    let acceptableThemes = allThemes;
+    const allowedThemeKeys = (config.AllowedThemes || '').split(',').filter(String);
```

Comment:
I'm not familiar with passing `String` into a filter. Does that just filter out non-empty entries like `Boolean` or one of the others would also do?